test/test_es_utils.py

```diff
@@ -1,7 +1,95 @@
 import pytest
+from unittest import mock
 from dcicutils.qa_utils import timed
 from dcicutils.ff_utils import get_es_metadata
-from dcicutils.es_utils import create_es_client, execute_lucene_query_on_es, get_bulk_uuids_embedded
+from dcicutils.es_utils import create_es_client, execute_lucene_query_on_es, get_bulk_uuids_embedded, ElasticSearchServiceClient
+
+
+class TestElasticSearchServiceClient:
+
+    @staticmethod
+    def mock_update_es_success(DomainName, ElasticsearchClusterConfig):
+        return {
+            'ResponseMetadata': {
+                'HTTPStatusCode': 200
+            }
+        }
+
+    @staticmethod
+    def mock_update_es_fail(DomainName, ElasticsearchClusterConfig):
+        return {
+            'ResponseMetadata': {
+                'HTTPStatusCode': 403
+            }
+        }
+
+    @staticmethod
+    def mock_update_es_bad_response(DomainName, ElasticsearchClusterConfig):
+        return {
+            'something_else': {
+                'blah': 403
+            }
+        }
```

Comment:
The real meat of this test is not just "does it return this?" but "is this function called with the right args?" so you need to tell it what args are the ones that should provoke this response.  (Same for the above mocks, I was too lazy to illustrate all of them.)

```suggestion
    @staticmethod
    def make_mock_update_es_bad_response(expected_domain, expected_config):
        def mock_update_es_bad_response(DomainName, ElasticsearchClusterConfig):
            assert expected_domain == DomainName
            assert expected_config == ElasticsearchClusterConfig
            return {
                'something_else': {
                    'blah': 403
                }
            }
```